packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -88,57 +89,62 @@ const getAccountInfo = async (request: Request<MessageTypes.GetAccountInfo>) =>
         } as const);
     }
 
-    // for the first page of txs, payload.page is undefined, for the second page is 2
-    const page = payload.page ? payload.page - 1 : 0;
-    const pageSize = payload.pageSize || 10; // TODO(vl): change to 25
+    const getTransactionPage = async (txIds: string[]) => {
```

Comment:
Basically no changes here to the previous implementation. All I did was
 - move getting of tx history into function so it can be called conditionally
 - call it only if details is `txs`